vhd/packer/windows-vhd-builder.json

```diff
@@ -117,17 +117,26 @@
         {
             "type": "powershell",
             "inline": [
+                "curl.exe -L https://go.microsoft.com/fwlink/?LinkID=394789 -o guestagent.msi",

```

Comment:
Does this link always point to the latest guest agent?
If not we can't keep this here and should add a comment or something.
Actually, we probably do not want to keep this in here regardless...

Reply:
This is a mitigation until the guest agent fix is rolled out.